scripts/install_platform_driver_configs.py

```diff
@@ -36,15 +36,15 @@
 # under Contract DE-AC05-76RL01830
 # }}}
 
-import gevent
-from volttron.platform.auth import AuthFile, AuthEntry
-from volttron.platform.vip.agent import Agent
+import os
+import glob
+
 from volttron.platform.agent.known_identities import CONFIGURATION_STORE, PLATFORM_DRIVER
 from volttron.platform import get_address
 from volttron.platform.keystore import KeyStore
+from volttron.platform.agent.utils import execute_command
+from volttrontesting.fixtures.volttron_platform_fixtures import build_wrapper
```

Comment:
It does not make sense to use platform wrapper in this

Reply:
Yes, removing build_wrapper and using the build_agent in utils from vip.utils instead.